src/modules/Writers.jl

```diff
@@ -142,7 +144,15 @@ function render(io::IO, mime::MIME"text/plain", node::Expanders.DocsNode, page,
                 """)
             end
         end
-        print(io, "</ul>")
+        print(io, "</ul>\n\n")
+
+        # we print a small notice if we are not displaying all the methods
+        nhidden = length(methodnodes)-length(ms)
+        if nhidden == 1
+            print(io, "_1 method is not being displayed._\n\n")
+        elseif nhidden > 1
+            print(io, "_$(nhidden) methods are not being displayed._\n\n")
+        end
```

Comment:
:+1: in general, but it did have is/are as well and then it would become a bit hard to read (not an issue after I rephrased it). I would drop the whitespace though, to make it more compact.